Dockerfiles/ocp4_content

```diff
@@ -8,10 +6,47 @@ RUN microdnf -y install cmake make git /usr/bin/python3 python3-pyyaml python3-j
 
 COPY . .
 
-RUN ./build_product --datastream-only --debug ocp4 rhcos4 eks
+# Enable the default.profiles as they maintain a list rules to be added to the datastream
+RUN find . -name "default\.profile" -exec sed -i 's/\(documentation_complete: \).*/\1true/' '{}' \;
+
+# Enable only certain profiles on ppc64le and s390x
+RUN if [ "$(uname -m)" == "x86_64" ]; then \
+    echo "Building OpenShift and RHCOS content for x86_64"; \
+    else echo "Building OpenShift content for $(uname -m)" && \
+        find . -name "*.profile" -exec sed -i 's/\(documentation_complete: \).*/\1false/' '{}' \; && \
+        sed -i 's/\(documentation_complete: \).*/\1true/' \
+        products/ocp4/profiles/pci-dss-node-3-2.profile \
+        products/ocp4/profiles/pci-dss-3-2.profile \
+        products/ocp4/profiles/pci-dss-node.profile \
+        products/ocp4/profiles/pci-dss.profile \
+        products/ocp4/profiles/cis-node.profile \
+        products/ocp4/profiles/cis.profile \
+        products/ocp4/profiles/cis-node-1-4.profile \
+        products/ocp4/profiles/cis-1-4.profile \
+        products/ocp4/profiles/cis-node-1-5.profile \
+        products/ocp4/profiles/cis-1-5.profile \
+        products/ocp4/profiles/moderate-node.profile \
+        products/ocp4/profiles/moderate.profile \
+        products/ocp4/profiles/moderate-node-rev-4.profile \
+        products/ocp4/profiles/moderate-rev-4.profile; \
+    fi
+
+
+# OCPBUGS-32794: Ensure stability of rules shipped
+# Before building the content we re-enable all profiles as hidden, this will include any rule selected
+# by these profiles in the data stream without creating a profile for them.
+RUN grep -lr 'documentation_complete: false' ./products | xargs -I '{}' \
+    sed -i -e 's/\(documentation_complete: \).*/\1true/' -e '/documentation_complete/a hidden: true' {}
```

Comment:
This is not needed, since the profiles were not disabled in the first place.

Reply:
what's a good catch, it should be put in non-x86 build